services/schedule_create.proto

```diff
@@ -90,4 +100,33 @@ message ScheduleCreateTransactionBody {
    * given
    */
   AccountID payerAccountID = 4;
+
+  /**
+   * An optional timestamp for specifying when the transaction should be evaluated for execution and then expire.
+   * Defaults to 30 minutes after the transaction's consensus timestamp.
+   *
+   * Note: This field is unused and forced to be unset until Long Term Scheduled Transactions are enabled - Transactions will always
+   *       expire in 30 minutes if Long Term Scheduled Transactions are not enabled.
+   */
+  Timestamp expirationTime = 5;
```

Comment:
We should use snake_case for new fields to provide better compatibility across languages.

Reply:
the hip uses camel case, since all the existing fields use it what benefit would be to changing now? I can change it, just curious.

Reply:
It's the standard we [agreed](https://github.com/hashgraph/hedera-protobufs/pull/13) upon for the protobuf repo. We just haven't had time to convert all the existing fields. This is also Google's [recommendation](https://developers.google.com/protocol-buffers/docs/style).

Reply:
done